node/network/availability-distribution/src/pov_requester/mod.rs

```diff
@@ -17,71 +17,31 @@
 //! PoV requester takes care of requesting PoVs from validators of a backing group.
 
 use futures::{FutureExt, channel::oneshot, future::BoxFuture};
-use lru::LruCache;
 
 use polkadot_subsystem::jaeger;
 use polkadot_node_network_protocol::{
-	peer_set::PeerSet,
 	request_response::{OutgoingRequest, Recipient, request::{RequestError, Requests},
 	v1::{PoVFetchingRequest, PoVFetchingResponse}}
 };
 use polkadot_primitives::v1::{
-	AuthorityDiscoveryId, CandidateHash, Hash, SessionIndex, ValidatorIndex
+	CandidateHash, Hash, ValidatorIndex,
 };
 use polkadot_node_primitives::PoV;
 use polkadot_subsystem::{
-	ActiveLeavesUpdate, SubsystemContext, ActivatedLeaf,
+	SubsystemContext,
 	messages::{AllMessages, NetworkBridgeMessage, IfDisconnected}
 };
-use polkadot_node_subsystem_util::runtime::{RuntimeInfo, ValidatorInfo};
+use polkadot_node_subsystem_util::runtime::RuntimeInfo;
 
 use crate::error::{Fatal, NonFatal};
 use crate::LOG_TARGET;
 
-/// Number of sessions we want to keep in the LRU.
-const NUM_SESSIONS: usize = 2;
-
-pub struct PoVRequester {
-	/// We only ever care about being connected to validators of at most two sessions.
-	///
-	/// So we keep an LRU for managing connection requests of size 2.
-	/// Cache will contain `None` if we are not a validator in that session.
-	connected_validators: LruCache<SessionIndex, Option<oneshot::Sender<()>>>,
-}
+pub struct PoVRequester {}
```

Comment:
I think we can get rid of this truct now.

Reply:
Fixed in e4cff40.